### ISSUE TITLE
Cannot attach equipment from win11

### ISSUE THREAD

**Issue by bemo47** [barbalion/usbip-win-client#11]
Hi,
I have installe usbip on the Pi4 and succeed to bind one usb equipment, a usb key for test.
On the Win11 i laso installed your solution, and succeeded to list the binded elements on the Pi4 from the Win11.
But when i launch the attach command i get "failed to attach"
I am not expert enough to understand what happens here, and what i did wrong...
Hope i can get some help.
![image](https://user-images.githubusercontent.com/44326240/149682545-3c4ccf29-c3e6-4ccc-af1d-3df057849999.png)

**Comment by renwar99** [comment]
try using sudo when you run usbipd on p4, i had the same issue , win11 starts and detects it but will never connect.

**Comment by bemo47** [comment]
Unfortunately that's what i did, all commands done with sudo on pi side.

**Comment by renwar99** [comment]
i think i installed the usbip drivers for WSL , I am not sure if that changed anything,. but what i know you have to enable unsigned drivers. and after i run the usbipd with sudo on pi4 and it just worked.  i had the exact same issue last night. i was testing and testing and it was not working. hence i found this thread. the following is my drive config and the pi4 interface 
![image](https://user-images.githubusercontent.com/34113942/149740919-da2df8de-a27a-42ac-9b1e-aaea12152495.png)
![image](https://user-images.githubusercontent.com/34113942/149741855-dc632a54-a41d-4e2f-979a-89cbf4cc0404.png)


[Edit]

Sorry and one more thing: add this to your modules... not sure it it helps:

![image](https://user-images.githubusercontent.com/34113942/149742885-ae21beec-d445-4a20-8144-906f43b198ee.png)

**Comment by bemo47** [comment]
u mean enable unsigned drivers on pi side or on windows side ?

**Comment by renwar99** [comment]
on windows 11 side. (normally unsigned drivers have a yellow triangular shape next to the driver icon. this means windows does not recognize or allow it in some cases.). and note this comes with some risk. try to see what are the risks. anyways here is the how to:

https://www.howtogeek.com/167723/how-to-disable-driver-signature-verification-on-64-bit-windows-8.1-so-that-you-can-install-unsigned-drivers/

**Comment by bemo47** [comment]
ok, so now i am in test mode but service not running and attach still failed
![image](https://user-images.githubusercontent.com/44326240/149746473-5ce6a42a-9c7c-4e49-bb2f-71626bc99ad7.png)

but at the end of the install.cmd, since the start i have this error
![image](https://user-images.githubusercontent.com/44326240/149746688-11794778-1f65-4059-aaf3-29c4386bba5e.png)

**Comment by renwar99** [comment]
What i done is deleting the service coupe of times and start over fresh(i mean : delete everything unzip the original and start again from there ). in windows here is how to delete the service:

From Admin-Powershell-command

sc.exe delete "USBIPService"

Note: once it says the service is deleted , you actually have to go to services and stop it. then it will be deleted.
also this version of usbip is trying to make the usbip an automated process. it has a lot of issues of detection. I will further debug tonight and see if i can find solutions for the common problems.


[Edit]


also reinstall the drivers if you can. delete all of them and install them from the install script

**Comment by bemo47** [comment]
yes, not easy, i deleted the service, then restarted, then install.cmd again with install service, but service stays "paused"....
![image](https://user-images.githubusercontent.com/44326240/149750361-311ab923-95d7-4ccd-a473-7c0726de5057.png)

**Comment by bemo47** [comment]
even unsinstalling all and reinstalling, same
![image](https://user-images.githubusercontent.com/44326240/149750873-34fd9665-50ac-45a3-b5c2-d25a2e4831ce.png)

**Comment by renwar99** [comment]
You sure you deleted the service? Go to services and check the old service status , make sure it is deleted before you start fresh

**Comment by barbalion** [comment]
Unfortunately, I don't have Win11 in my possession to test at the moment.
Try rebooting the system before fresh install. Deleting services can leave traces in the running system.

**Comment by bemo47** [comment]
Yes i did, deleted service, uninstall all, reboot, still in test mode, install again, test and same...
